docs/contacts/preference_center.rst

```diff
@@ -4,5 +4,192 @@ Preference center
+
+.. vale off
+
+Contact Segments
+****************
+
+.. vale on
+
+.. image:: images/segments.png
+    :align: center
+    :alt: Screenshot of Segments
+
+|
+
+Use the Segments tab to add or remove a Contact from a Segment. Segments are used as a source for Campaigns and Emails. Any Contact in a particular Segment will be part of a Campaign that has that Segment as the source. You can also use a standalone Email manually to a Segment. If a User has opted out of a Segment it will no longer receive Campaign actions or messages sent to that Segment.
+
+.. vale off
+
+Contact's Unsubscribe Email Preferences
+***************************************
+
+.. vale on
+
+.. image:: images/email-unsubscribe-settings.png
+    :align: center
+    :alt: Screenshot of Email unsubscribe
+
+|
+
+The Contact's preferences can be presented to the User in the unsubscribe page by selecting "Show Contact preference settings" in the Email configuration. You may also choose to hide or show different Segments of the User preferences. If any of these areas is set to no, it hides it from the contact's personal preferences page. The default unsubscribe message is shown if the preference setting option is set to no.
+
+.. image:: images/unsubscribe.png
+    :align: center
+    :alt: Screenshot of Unsubscribe
+
+|
+
 Customize preference center
-***************************
\ No newline at end of file
+===========================
+
+It's possible to customize the personal Preference Center/unsubscribe page, edit text labels, format, and apply Themes using the landing page builder.
+
+.. vale off 
+
+Creating a Preference Center Landing Page
+*****************************************
+
+.. vale on
+
+When creating/editing a landing page, there is a toggle switch labeled Is *Preference Center*. If selected, the page will be marked as a preference center landing page.
+
+When this page is configured as a preference center in a Mautic Email, recipients will be shown the page when clicking on the ``{unsubscribe_url}`` link. It also shows or hides the Preference Center slots in the builder.
+
+.. image:: images/pref1.png
+    :align: center
+    :alt: Screenshot of Preference
+
+|
+
+Builder slots
+*************
+
+These slots in the builder are used to customize the page:
+
+.. image:: images/pref2.png
+    :align: center
+    :alt: Screenshot of Preference
+
+|
+
+Preference tokens
+******************
+
+Optionally, you can use :doc:`tokens</configuration/variables>` to insert the different slots. Keep in mind that if you use :doc:`tokens</configuration/variables>`, you lose the ability to customize the labels and styles of the slots because it uses the default ones.
+
+
+.. image:: images/pref3.png
+    :align: center
+    :alt: Screenshot of Preference
+
+|
+
+See the :doc:`VARIABLES</configuration/variables>` page for a full list of tokens.
```

Comment:
Let's link directly to the section for the preference centre tokens using the heading?

Reply:
I am kinda confused here. Please can you elaborate?

Reply:
Instead of just dumping at the top of the page using https://mautic-documentation.readthedocs.io/en/latest/configuration/variables.html let's take them to where they need to be, eg: https://mautic-documentation.readthedocs.io/en/latest/configuration/variables.html#preference-center-landing-page-tokens using the anchor heading.